packages/map-template/src/components/Directions/Directions.jsx

```diff
@@ -43,7 +58,19 @@ function Directions({ isOpen, onBack, directions }) {
 
             // Set the step index to be 0 in order to display the correct instruction on the map.
             directionsRenderer.setStepIndex(0);
+
+            originInfoElement.current.location = directions.originLocation;
```

Comment:
Is the useRef needed? Is it because the `<mi-location-info>` doesn't take an object? 

Is it possible to just pass down the `directions.originLocation` to the `<mi-lcoation-info>` element?

Reply:
@ammapspeople I tried to use the `directions.originLocation` directly on the `mi-location-info` element, but it doesn't seem to be possible. 🤔 Looks like the useRef is needed

Reply:
I see that we do the same in LocationDetails as well. All right 😄